### PR TITLE
chore: make all illustration images non draggable

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/16879/desktop-illustrations-shouldn-t-be-draggable

#### Testing instructions
Illustrations in app should not be draggable. This is most noticeable in an empty folder, where dragging the background image causes the drag-and-drop upload UI highlight to show up. But in general it's something that can happen by accident and looks weird.

Affected illustrations:
* room invite
* chat zero state
* files zero state (including empty folder)
* contact list zero state and contact invite

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

**Comment by lostfictions** [comment]
ugh ci failure is the result of desktop riding `icebear#dev` -- forgot about that. @anri-asaturov i think all new desktop installs are going to fail until the desktop translator pr is merged :(  my fault for forgetting we ride the branch instead of versioning.

**Comment by anri-asaturov** [comment]
Don't worry about that, it's a slow time and we just have released. Wednesday release will be from master anyway.

File: src/ui/chat/components/ChannelInvite.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ export default class ChannelInvite extends React.Component<ChannelInviteProps> {
         return (
             <div className={css('channel-invite', this.props.className)}>
                 <div className="invite-content">
-                    <img src="./static/img/illustrations/room-invite.svg" />
+                    <img src="./static/img/illustrations/room-invite.svg" draggable={false} />
```
Comment (anri-asaturov) [inline src/ui/chat/components/ChannelInvite.tsx]: does it have to be `{false}`? it's a html attribute

Comment (suprko) [reply src/ui/chat/components/ChannelInvite.tsx]: as in, why can't it be `'false'`? JSX expects a boolean

Comment (anri-asaturov) [reply src/ui/chat/components/ChannelInvite.tsx]: right, typescript sacrifice :)

Comment (anri-asaturov) [reply src/ui/chat/components/ChannelInvite.tsx]: nope, react unserstands `draggable="false"` fine

Comment (lostfictions) [reply src/ui/chat/components/ChannelInvite.tsx]: oh that's weird, i thought it usually didn't actually work with string attributes... are you sure it's not doing some weird coercion to falsy?

Comment (anri-asaturov) [reply src/ui/chat/components/ChannelInvite.tsx]: well ="true" works, so i guess no, it just knows how to handle it, OR, more likely, it just passes the string and DOM knows what to do with it. Although it does seem better type wise to use boolean values.

**Comment by ebilg** [comment]
can be merged

**Pull request merged**